housekeeper/cli/get.py

```diff
@@ -143,17 +144,18 @@ def files_cmd(
     compact: bool,
 ):
     """Get files from database"""
-    store: Store = context.obj["store"]
-    file_objs = store.get_files(bundle_name=bundle, tag_names=tag_names, version_id=version_id)
-    template = schema.FileSchema()
-    result = []
-    for file in file_objs:
-        result.append(template.dump(file))
-    if json:
-        click.echo(jsonlib.dumps(result))
-        return
-    console = Console()
-    console.print(get_files_table(result, verbose=verbose, compact=compact))
+    file_service: FileService = context.obj["file_service"]
+    output_service: OutputService = context.obj["output_service"]
+
+    output_service.verbose = verbose
+    output_service.compact = compact
+    output_service.json = json
+
+    local = file_service.get_local_files(bundle=bundle, tags=tag_names, version_id=version_id)
+    remote = file_service.get_remote_files(bundle=bundle, tags=tag_names, version_id=version_id)
+
+    output_service.log_file_table(files=local, header="Local files")
+    output_service.log_file_table(files=remote, header="Remote files")
```

Comment:
These 4 lines could be wrapped in a single method in a service, maybe the `FileService`.